src/styles/common/_tooltip.scss

```diff
@@ -11,12 +11,6 @@ $tooltip-width: 1;
     .data-point-highlighter {
         fill: $white;
         stroke-width: 2;
-        // &:after {
```

Comment:
was this intentional?

Reply:
No! Good catch.

Reply:
Oh, I take it back. I was cleaning up commented out code.

Reply:
Great!